### PR TITLE
Fix doc

### PR DESCRIPTION
**Pull request by Louis-Dupont**
3 ReadMe fixes
- model names (x2)
- setup_device documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,9 +320,6 @@ init_trainer()
 # Launch DDP on 4 GPUs'
 setup_device(multi_gpu=MultiGPUMode.DISTRIBUTED_DATA_PARALLEL, num_gpus=4)
```
Comment (shaydeci) [inline README.md:322]: Still need to instantiate the trainer when launched this way.

File: src/super_gradients/training/Computer_Vision_Models_Pretrained_Checkpoints.md [hunk 2]
```diff
@@ -11,25 +11,24 @@ All the available models are listed in the column `Model name`.
...
-| MobileNet V3_large-300 epochs | mobilenet_v3_large | ImageNet    | 224x224    | 74.52   | 91.92   | **0.67ms**                 | **1.11ms**                          | **2.42ms**                               | **2.71ms**                                        |           **1.76ms**            |
-| MobileNet V3_small            | mobilenet_v3_small | ImageNet    | 224x224    | 67.45   | 87.47   | **0.55ms**                 | **0.96ms**                          | **2.01ms** *                             | **2.35ms**                                        |           **1.06ms**            |
-| MobileNet V2_w1               | mobilenet_v2 ?     | ImageNet    | 224x224    | 73.08   | 91.1    | **0.46 ms**                | **0.89ms**                          | **1.65ms** *                             | **1.90ms**                                        |           **1.56ms**            |
+beit_base_patch16_224_imagenet.pth
```
Comment (shaydeci) [inline src/super_gradients/training/Computer_Vision_Models_Pretrained_Checkpoints.md:14]: think this slipped out.

**Review by shaydeci — CHANGES_REQUESTED** [review]

File: src/super_gradients/training/Computer_Vision_Models_Pretrained_Checkpoints.md (hunk 2 above)
Comment (Louis-Dupont) [reply src/super_gradients/training/Computer_Vision_Models_Pretrained_Checkpoints.md]: changed in dcde517e71600e4183edb385da5d897b30830e4c

File: README.md (hunk 1 above)
Comment (Louis-Dupont) [reply README.md:322]: changed in dcde517e71600e4183edb385da5d897b30830e4c

**Review by shaydeci — APPROVED** [review]
LGTM

**Pull request merged**